src/material/icon/icon-registry.ts

```diff
@@ -132,6 +133,7 @@ export class MatIconRegistry implements OnDestroy {
   constructor(
     @Optional() private _httpClient: HttpClient,
     private _sanitizer: DomSanitizer,
+    private readonly _errorHandler: ErrorHandler,
```

Comment:
Can you make this the last argument and make it `@Optional`? Technically changing the constructor signature is a breaking change for anyone extending the class

Reply:
Done